node/coinstacks/optimism/pulumi/index.ts

```diff
@@ -14,19 +14,29 @@ export = async (): Promise<Outputs> => {
       case 'daemon':
         return {
           ...service,
+          env: {
+            NETWORK: config.network,
+            SNAPSHOT: 'https://storage.googleapis.com/oplabs-goerli-data/goerli-bedrock.tar', // TODO FINAL SNAPSHOT ENDPOINT
```

Comment:
I suppose this is also to be updated today

Reply:
yep, all snapshot, image, and chain refs